Tests/TestingApps/PurchaseTesterSwiftUI/Shared/Extensions/Extensions.swift

```diff
@@ -59,8 +59,9 @@ extension Configuration.EntitlementVerificationMode {
 
     static let all: [Self] = [
         .disabled,
-        .informational,
-        .enforced
+        .informational
+        // Disabled for now:
```

Comment:
Linear is private but this is a public repo so I'm not sure that's super useful.

Reply:
Right... I just think this could raise more questions. Since we are going to be considering this feature `beta` for now, maybe we can mention something about this being disabled during beta? NABD though.

Reply:
I was thinking that since this doesn't compile, if you try to enable it you'll see the "unavailable" message saying that it will be available in a future release.

Reply:
Right, my concern is for those looking at the code (which would mostly be us), this can look cryptic down the line, as in "why is this disabled?" I think ideally there would be a reference here as for why it's disabled. Whether it's because the feature is in beta or something else.

Reply:
Ooh I see what you're saying, I'll improve the comment